src/app.js

```diff
@@ -247,8 +249,8 @@ class Messenger extends EventEmitter {
     } = message;
 
     if (this.options.emitGreetings && this.greetings.test(text)) {
-      const firstName = session.profile.first_name.trim();
-      const surName = session.profile.last_name.trim();
+      const firstName = session.profile && session.profile.first_name.trim() || '';
```

Comment:
This was to satisfy Flow's warning about `session.profile` being null. It never comes up in practice, but this does seem more robust and is probably a good idea.